database/N/NEFFEX/Fight Back: The Collection/Fight Back

```diff
@@ -0,0 +1,72 @@
+You're never gonna make it, you're not good enough
```

Comment:
Could you please change the `'` signs to proper apostrophe signs?  More info here: https://smartquotesforsmartpeople.com